app/views/matchups/_matchup.html.erb

```diff
@@ -59,7 +59,7 @@
             </div>
           </div>
 
-          <% unless matchup.done %>
+          <% unless matchup.done || 1 == 1 %>
```

Comment:
Why put this in? Is this just to temporarily always display it?

Reply:
This hides the button, since we don't have that feature